packages/api/src/command/medical/patient/__tests__/store-query-cmd.ts

```diff
@@ -0,0 +1,64 @@
+import * as uuidv7_file from "@metriport/core/util/uuid-v7";
+import { DocumentQueryProgress } from "@metriport/core/domain/document-query";
+import { PatientDiscovery, QueryProgress } from "@metriport/core/domain/query-status";
+import { makeProgress } from "../../../../domain/medical/__tests__/document-query";
+import { StoreQueryParams } from "../query-init";
+import { makePatientData } from "../../../../domain/medical/__tests__/patient";
+import { makePatientModel } from "../../../../models/medical/__tests__/patient";
+
+export const requestId = uuidv7_file.uuidv4();
+export const patient = { id: "patient-id", cxId: "cx-id" };
```

Comment:
I suggest we create actual UUIDs, could use `faker.string.uuid()` or just `uuidv7()` for even better test quality (closer to the reality).